### ISSUE TITLE
Disconnected haplotype with beta, was connected before

### ISSUE THREAD

**Issue by skoren** [marbl/verkko#44]
Chr5 of hg002 has a disconnected node (utig4-4083) on the maternal haplotype. See below image with the pre-release assembly from Dec on the left vs the beta assembly on the right:

<img width="2954" alt="Screen Shot 2022-02-02 at 10 03 44 AM" src="https://user-images.githubusercontent.com/923478/152194868-41cee065-c191-4df9-8119-bf840d74847c.png">

Tracking back the relevant nodes, it seems the issue comes up in the hifi-resolved graph, even though the input HiFi reads are identical. There is one node (utig1-61600 in the old, utig1-68891) that looks like it became a dead-end. 
<img width="2937" alt="Screen Shot 2022-02-02 at 11 42 57 AM" src="https://user-images.githubusercontent.com/923478/152197923-b61dd581-46f6-424c-889b-287fd90510de.png">

There are 47 reads in utig1-68891 which are in utig1-61600 and utig1-61598 in the old graph. Listed below:
```
m54329U_201103_231616/124979520/ccs
m54329U_201103_231616/125307390/ccs
m54329U_201103_231616/129960611/ccs
m54329U_201103_231616/162072659/ccs
m64011_190714_120746/100860855/ccs
m64011_190714_120746/132907669/ccs
m64011_190714_120746/134742206/ccs
m64011_190714_120746/172755081/ccs
m64011_190714_120746/180027541/ccs
m64011_190714_120746/21299803/ccs
m64011_190714_120746/67767043/ccs
m64011_190728_111204/16319392/ccs
m64011_190728_111204/32507695/ccs
m64011_190728_111204/61932073/ccs
m64011_190728_111204/98502297/ccs
m64011_190830_220126/102957179/ccs
m64011_190830_220126/123797740/ccs
m64011_190830_220126/134809327/ccs
m64011_190830_220126/148703224/ccs
m64011_190830_220126/149227846/ccs
m64011_190830_220126/159187843/ccs
m64011_190830_220126/166921258/ccs
m64011_190830_220126/169214396/ccs
m64011_190830_220126/172295046/ccs
m64011_190830_220126/19203472/ccs
m64011_190830_220126/79235545/ccs
m64011_190901_095311/111675710/ccs
m64011_190901_095311/123208997/ccs
m64011_190901_095311/28772647/ccs
m64011_190901_095311/39388443/ccs
m64012_201024_012808/21300262/ccs
m64012_201024_012808/37554571/ccs
m64012_201024_012808/52299945/ccs
m64012_201024_012808/59900338/ccs
m64012_201024_012808/67568265/ccs
m64012_201024_012808/83101870/ccs
m64012_201024_012808/86508091/ccs
m64012_201025_074858/120720047/ccs
m64012_201025_074858/132123423/ccs
m64012_201025_074858/146997854/ccs
m64012_201025_074858/2031793/ccs
m64012_201025_074858/59901372/ccs
m64012_201025_074858/75499961/ccs
m64076_201013_225902/100534931/ccs
m64076_201013_225902/131793924/ccs
m64076_201016_191536/124914513/ccs
m64076_201016_191536/169934935/ccs
```

**Comment by skoren** [comment]
This is similar to bonobo issue (#27) and is due to minor changes in corrected reads/input order. Will work on tracking down.

**Comment by skoren** [comment]
This was due to a bug in correction parameters, fixed in #635d4af5a75d93ebdeb93dc4f1a5a7a78b313073
